src/map/packets/message_standard.h

```diff
@@ -31,49 +31,69 @@
-    CannotInvite            = 23, /* You cannot invite that person at this time. */
-    WaitLonger              = 38, /* You must wait longer to perform that action. */
-
-    DiceRoll                = 88, /* Dice Roll! <player> rolls <roll>. */
-    Examine                 = 89, /* <name> examines you. */
-    /* Linkshells */
-    LinkshellUnavailable    = 112, /* The linkshell name you entered is already in use or otherwise unavailable. */
-    TellNotReceivedOffline  = 125, /* Your tell was not received.  The recipient is currently away. */
-    ThrowAway               = 180, /* You throw away <item>. */
-    TellNotReceivedAway     = 181, /* Your tell was not received.  The recipient is currently away. */
-    ItemEx                  = 220, /* You cannot possess more than one of that item. */
-    BlockingAid             = 225, /* Target is currently blocking outside magical assistance */
-    Sell                    = 232, /* You sell <item>. */
-    SellToShop              = 233, /* You sell <item> to the shop. */
-    LevelSyncWarning        = 235, /* Warning! This is a Level Sync party ... */
-    CannotInviteLevelSync   = 236, /* You cannot invite that person at this time. This player is either undergoing Level Sync... */
-    CannotJoinLevelSync     = 237, /* You cannot join this party.  You are either underoing Level Sync... */
-    Compass                 = 239, /* The compass reads: ...*/
-    CannotHere              = 256, /* You cannot use that command in this area. */
-
-    /* Style Lock*/
-    HeadgearShow            = 260,
-    HeadgearHide            = 261,
-    StyleLockOn             = 267, /* Style lock mode enabled. */
-    StyleLockOff            = 268, /* Style lock mode disabled. */
-    StyleLockIsOn           = 269, /* Style lock mode is enabled. */
-    StyleLockIsOff          = 270, /* Style lock mode is disabled. */
-
-    /* Trusts */
-    TrustCannotJoinParty    = 265, /* You are unable to join a party whose leader currently has an alter ego present. */
-    TrustCannotJoinAlliance = 266, /* You are unable to join an alliance whose leader currently has an alter ego present. */
-    TrustCannotLFP          = 296, /* You cannot use Trust magic while seeking a party. */
-    WaitParty               = 297, /* While inviting a party member, you must wait a while before using Trust magic. */
-    TrustLimit              = 298, /* You have called forth your maximum number of alter egos. */
-    TrustSame               = 299, /* That alter ego has already been called forth. */
-    TrustEnmity             = 300, /* You cannot use Trust magic while having gained enmity. */
-    TrustSoloOrLeader       = 301  /* You cannot use Trust magic unless you are solo or the party leader. */
+    // Used as a sentinal value. This should not be used as part of a packet.
+    Unknown = -1,
+
+    InvitationDeclined             = 11, /* Your invitation was declined */
```

Comment:
I realize a past author did this, but this seems a good chance to clean up that comment style.

```cpp
// This is a single line comment
```
```cpp
/* 
this is a
multi-line or "block"
comment
*/
```

Reply:
Is this the preferred style?

I tend to like doxygen style comments since they can also be parsed by many tools. But I'm happy to do whatever is consistent with the project.

--

Also is there a preferred naming convention? It seems to be a mix of `camelCase` , `CamelCase`, `snake_case`, and `everythinglowercase`.

I tend to try to apply Google's C++ style whenever possible, since it's documented and widely accepted. But the code style in DSP seems a bit inconsistent across functions and files.

Reply:
style guide has some of it. other parts need new style guide entries. We defo don't follow google conventions here. We have different function styles to indicate different things.

UPPER_SNAKE_CASE for ENUM
UpperCamelCase for namespaced functions and classes
lowerCamelCase for pure lua functions (not in cpp at all)

If you want to pitch using a tool that requires those, talk to the team on discord and if we're on board with it, we can go the other route instead and convert them all to allow for that tool.Right now though most files are using a lot of non-block comments

Reply:
If you want to pitch using a tool that requires those block comments, talk to the team on discord and if we're on board with it, we can go the other route instead and convert them all to allow for that tool.Right now though most files are using a lot of non-block comments

Reply:
I'm not too concerned about using a tool. Some IDEs just pick up doxygen style comments by default.

However I'm happy to just match the comment style you mentioned. Will update.

Reply:
Updated to match the recommended style.